doc/source/User_guide/pyaedt_extensions_doc/project.rst

```diff
@@ -0,0 +1,49 @@
+Import Nastran
+==============
+
+Import Nastran or STL file in any 3D modeler application. The user can also preview the imported file and decimate it before import it.
+
+User can access the extension from the icon created in the Automation tab through the extension manager.
```

Comment:
```suggestion
The extension can be accessed from the icon created in the Automation tab through the extension manager.
```

Reply:
You can access the extension from the icon created on the **Automation** tab using the Extension Manager.

Use bold for UI elements. Capitalize "Extension Manager" as its a proper noun of an Ansys component.